### ISSUE TITLE
Inference Using run_tfilte.py : Getting low score prediction also some objects are still not detected ?

### ISSUE THREAD

**Issue by Kapil-23** [google/automl#1156]
Generate TF-Lite model using below script : 

`!python model_inspect.py --runmode=saved_model --model_name=efficientdet-d1 \
  --ckpt_path=AUTO_ML-50-30k --saved_model_dir=savedmodeldir-50-30k \
  --tflite_path=efficientdet-d1-TF-50EP-30k.tflite \
  --hparams=voc_config.yaml`

I have trained efficientdet d1 (640x640) by setting --num_epochs=50 epochs.
In some images model is able to  detect objects.
But In most of the other images model is not performing well.
Also accuracy on detection remains near by to 70 %.
Is there any parameter need to be included while converting the model or any preprocessing need to be done ?

One more query:
At the time of inference i am manually resizing the image by keeping aspect ratio and pasting the image to 640x640 canvas by gray padding. 
Further i feed this image to model.
FYI Model :
Input : [1,640,640,3]
Ouput : [1,100,7]
Is this the correct way ?
